content/blog/en/version-6-dojo.md

```diff
@@ -0,0 +1,295 @@
+---
+title: Announcing Dojo 6
+date: 2019-07-30T08:00:00.000Z
+author: Anthony Gubler
+---
+
+Since the first major Dojo release last year, we have been working to refine the features and patterns to make Dojo an even more efficient framework for building applications with TypeScript and modern web APIs.
+
+In a world of semantic versioning where even minor breaking changes require a new version number, it is challenging to know when a new version is substantial. Today, we’re excited to announce version 6, our most ambitious set of improvements since the Dojo 2.0 release.
+
+Version 6 of Dojo brings many new features and changes to substantially improve the development experience by reducing boilerplate, increasing flexibility, and further improving performance.
+
+![The image for the blog](assets/blog/version-6-dojo/featured.png)
+<!-- more -->
+
+## Function Based Widgets and Middleware
+
+We are very excited to introduce function-based widgets and middleware, the next evolution for creating and working with widgets in Dojo. Function-based widgets and middleware offer an alternative API to the existing class-based APIs (metas, decorators, mixins). A single API for both widgets and middleware helps improve developer ergonomics and reduce the complexity and boilerplate compared to previous releases of the framework, making it even easier to start writing Dojo applications.
+
+What is middleware? Middleware is the singular concept designed to replace all existing supplemental widget patterns, mixins, metas and decorators. A natural progression from the functional and reactive meta API, middleware provides a mechanism that not only facilitates working with imperative and asynchronous APIs reactively, but can get composed with other middleware and also affect a widget’s property interface.
+
+The core primitive for working with function-based widgets and middleware is a new function, `create`, provided by the `vdom` module. The `create` function gets used for defining middleware and properties and returns a factory for creating either widgets or middleware.
+
+> src/MyWidget.tsx
+```tsx
+import { create, tsx } from '@dojo/framework/core/vdom';
+
+interface MyWidgetProperties {
+	label: string;
+}
+
+const render = create().properties<MyWidgetProperties>();
+
+export default render(function MyWidget({ properties, children }) {
+  const { label } = properties();
+	return (
+		<div>
+			<span>{label}</span>
+			<div>{children()}<div>
+		</div>
+	);
+});
+```
+
+As mentioned, the middleware design supports composition in order to create advanced custom functionality. The majority of middlewares will build on the set of core middlewares which provide hooks into Dojo’s rendering engine. For more information on the core middleware please see the [Dojo reference guide](/learn/middleware/core-render-middleware).
```

Comment:
A small change to use active voice instead of passive here.

```suggestion
The middleware design supports composition in order to create advanced custom functionality. The majority of middlewares will build on the set of core middlewares which provide hooks into Dojo’s rendering engine. For more information on the core middleware please see the [Dojo reference guide](/learn/middleware/core-render-middleware).
```